subgraph/src/public-lock.ts

```diff
@@ -399,3 +406,22 @@ export function handleLockMetadata(event: LockMetadataEvent): void {
     lock.save()
   }
 }
+
+export function handleReferrerFees(event: ReferrerFeeEvent): void {
+  const lock = Lock.load(event.address.toHexString())
+
+  if (lock) {
+    const referrerAddress = event.params.referrer.toHexString()
+
+    let referrerFee = ReferrerFee.load(referrerAddress)
+
+    if (!referrerFee) {
+      referrerFee = new ReferrerFee(referrerAddress)
+    }
```

Comment:
@julien51 Here we either load an existing `ReferrerFee` by `referrer address` address or create a new one if it doesn't exist.